src/main/java/DilemmaDetector/Consequences/CollisionConsequencePredictor.java

```diff
@@ -23,15 +23,25 @@ public CollisionConsequencePredictor(IConsequenceContainer consequenceContainer,
         this.model = model;
     }
 
+    private boolean isPedestrian(Actor victimActor, Living_entity victim){
+        return victimActor.getEntityName().equals(victim.getOwlIndividual().getIRI().toString());
```

Comment:
Sprawdzam czy nazwa aktora (może być np. pedestrainem, lub vehiclem) jest równa nazwie ofiary. Jeśli tak to muszą być pieszym. To trochę brzydkie ale działa a nie miałem pomysłu jak to zrobić inaczej 
![image](https://user-images.githubusercontent.com/47008811/98415352-73c19780-207d-11eb-8a32-6c901b3240ce.png)

Reply:
tutaj dodaj komentarz z krótkim i treściwym wyjaśnieniem dlaczego to działa, jeżeli sie model skomplikuje, to będzie problem